src/windows/models/emoji_model.py

```diff
@@ -73,9 +74,10 @@ def update_model(self, clear=True):
         if clear:
             self.model_paths = {}
             self.model.clear()
+            self.emoji_groups.clear()
 
         # Add Headers
-        self.model.setHorizontalHeaderLabels([_("Thumb"), _("Name")])
+        self.model.setHorizontalHeaderLabels([_("Name")])
```

Comment:
There's no reason to maintain separate "Thumb" and "Name" columns. The Icon and the text for a column are set as different roles, meaning both items can occupy the same column. Actually, for reasons I'll get into it makes a lot of sense to put as _much_ as possible into a single column.